Ermöglichung eines sofortigen Vereinsaustritts

renemeye opened the pull request:
Dies ist ein Antrag zur Änderung von Satzung und Beitrags Ordnung zur **Mitgliederversammlung am 14.10.2015**
_Der Pull-Request darf natürlich nur nach Abstimmung durch den verein angenommen oder abgelehnt werden._

**Zur Begründung:**
Der Verein hat einmal beschlossen, den Vereinsaustritt erst zum Ende des folgenden Quartals zu ermöglichen, um die Mitgliedsbeiträge innerhalb der Kündigungsfrist der Vereinsräume sicherzustellen.
Mittlerweile hat der Verein genug Rücklagen um einen plötzlichen Massenaustritt der Mitglieder daraus abzufedern. Daher ist die Notwendigkeit der Regelung hinfällig.

Die aktuelle Regelung hat zu mehreren Problemen geführt. Mitglieder die bereits ihren Austrittswunsch geäußert hatten, mussten auch Monate später noch für ein weiteres Quartal aufkommen obwohl sie im Regelfall nichts mehr mit dem Vereinsleben zu tun haben. Dies wirkt sehr negativ auf das austretende Mitglied und wirft ein schlechtes Licht auf den Verein. Der Vorstand war auch in Fällen von finanzieller Not nicht in der Lage diesen Beitrag zu erlassen.
Ein Mitglied welches einen Austrittswunsch äußert, beendet die Mitgliedschaft erst 4-6 Monate später. Ein Mitglied erst nach 4-6 Monaten aus allen Systemen zu nehmen um entsprechende Firsten einzuhalten ist ein erheblicher Mehraufwand in der Mitgliederverwaltung.

**Neue Konstruktion:**
- Ein Vereinsaustritt ist sofort möglich. (Dies spart viel Verwaltungsaufwand.)
- Die Mitgliedsbeiträge sind nicht mehr Monats- sondern Quartalsbeiträge. Somit kann keine Rückforderung für Folgemonate erstritten werden.
- Eingezogene Mitgliedsbeiträge werden grundsätzlich nicht zurückerstattet.
- Somit ist die Finanzierung des aktuellen Quartals sichergestellt.

Detaillierte Begründung an den einzelnen Zeilen.

Commit:
Ermöglichung eines sofortigen Vereinsaustritts

renemeye commented on Beitrags_und_Mahnordnung.md:
```diff
@@ -1,18 +1,16 @@
-in der Fassung vom 06.05.2012
-
-Im Zuge der Gründungsversammlung am 06.05.2012 wurde die Beitrags- und Mahnordnung in der vorliegenden Fassung von der Gründungsversammlung beschlossen
+in der Fassung vom 14.10.2015
```
Aktualisierung des Datums und Streichung der obsoleten Einführung.

renemeye commented on Beitrags_und_Mahnordnung.md:
```diff
@@ -1,18 +1,16 @@
 2. Für den Beitritt wird eine Beitrittsgebühr in Höhe von 10,00 € erhoben, die mit dem ersten Beitrag fällig wird.
 
-3. Der Monatsbeitrag beträgt für aktive Mitglieder 30,00 €.
+3. Der Quartalsbeitrag beträgt für aktive Mitglieder 90,00 €.
```
- Der Mitgliedsbeitrag bleibt gleich. Auch die Einzugsart bleibt unverändert.
- Änderung von Monatsbasierten auf Quartalsbasierte Mitgliedsbeiträge. Dadurch ist eine mögliche Rückforderung von Folgemonaten ausgeschlossen.

bastinat0r replied:
Dadurch wird ein Beitritt zum Quartalsende sehr ungünstig. Wenn jemand im  März beitritt, muss er für Januar und Februar noch Beitrag zahlen.

renemeye replied:
Das stimmt. Sollte hiermit gefixed sein: 3b7ecd1

renemeye commented on Beitrags_und_Mahnordnung.md:
```diff
@@ -1,18 +1,16 @@
+3. Der Quartalsbeitrag beträgt für aktive Mitglieder 90,00 €.
 
-4. Auf Wunsch eines Mitglieds wird ein ermäßigter Beitragssatz in Höhe von 10,00€ pro Monat gewährt. Dieser Wunsch kann jederzeit beim Schatzmeister für ein Kalenderjahr angezeigt werden. Alle Mitglieder sind angehalten, diese Regelung nur im Bedarfsfall zu nutzen.
+4. Auf Wunsch eines Mitglieds wird ein ermäßigter Beitragssatz in Höhe von 30,00€ pro Quartal gewährt. Dieser Wunsch kann rechtzeitig vor Beginn eines Quartals beim Schatzmeister für ein Kalenderjahr angezeigt werden. Alle Mitglieder sind angehalten, diese Regelung nur im Bedarfsfall zu nutzen.
```
- Gleiche Änderung auf Quartalsbasis.

UND
- Klarstellung der Formulierung "jederzeit"

renemeye commented on Satzung.md:
```diff
@@ -1,11 +1,11 @@
 *des Netz39 e.V. - Hackerspace Magdeburg*
 
-in der Fassung vom 24.01.2013
+in der Fassung vom 14.10.2015
```
Aktualisierung des Datums.

renemeye commented on Satzung.md:
```diff
@@ -1,11 +1,11 @@
 Präambel
 --------
-Im folgenden finden sich Satzungen und Ordnungen des Vereins Netz39 auf dem Stand vom 24.01.2013. In diesem Dokument wird der Einfachheit halber ein generisches Maskulinum (z.B. "ein Mitglied") verwendet. Damit sind jedoch stets alle Geschlechter gemeint. An Stellen, an denen schriftliche Kommunikation gefordert wird, ist E-Mail stets mit eingeschlossen.
+Im folgenden finden sich Satzungen und Ordnungen des Vereins Netz39 auf dem Stand vom 14.10.2015. In diesem Dokument wird der Einfachheit halber ein generisches Maskulinum (z.B. "ein Mitglied") verwendet. Damit sind jedoch stets alle Geschlechter gemeint. An Stellen, an denen schriftliche Kommunikation gefordert wird, ist E-Mail stets mit eingeschlossen.
```
Aktualisierung des Datums.

renemeye commented on Satzung.md:
```diff
@@ -43,7 +43,7 @@ Im folgenden finden sich Satzungen und Ordnungen des Vereins Netz39 auf dem Stan
 
 3. Die Mitgliedschaft endet
- -	durch freiwillige Beendigung mit viermonatiger Frist zum Ende des Quartals durch schriftliche Erklärung gegenüber dem Vorstand;
+ -	durch freiwillige Beendigung ohne First durch schriftliche Erklärung gegenüber dem Vorstand;
```
Sofortiger Austritt wird damit ermöglicht.

bastinat0r replied:
-- First
++ Frist

renemeye committed:
Fixed Hürde zur Neuaufnahme von Mitgliedern

In der vorliegenden Version hätten Neumitglieder den gesamten Quartalsbeitrag bei Eintritt erbringen müssen. Dies ist hiermit gelöst.

renemeye committed:
Fixed typo

renemeye commented:
@bastinat0r Der von dir gefundene Typo ist hier gelöst: 68f4291

renemeye commented:
Dieser Änderungsantrag wurde auf der Mitgliederversammlung am 15.10.2015 einstimmig abgelehnt.
